andela/odin-ah-frontend
# #160452446 In-app  notification

#### What does this PR do?
 Add real-time in-app notification
#### Description of Task to be completed?
-> create reducer and actions for fetching in-app notifications
-> create reducer and actions for updating  notifications
-> integrate socket.io for real-time notification
-> create a dropdown for displaying notifications
-> add test for notification reducers
->  test for notification action

#### How should this be manually tested?

```
git checkout staging 
```
```
 git pull
```
```
git checkout ft-view-notification-160452446
```
```
npm install
```
Register Author's haven and login and you will see notification icon on the navbar
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/160452446

#### Screenshots (if appropriate)
<img width="695" alt="g" src="https://user-images.githubusercontent.com/29021978/49008269-aebab100-f16d-11e8-814e-770d07e4cb2a.png">

#### Questions:
Nil
